recipes-domu/domu-image-weston/files/meta-xt-prod-extra/recipes-graphics/gles-module/gles-user-module.inc

```diff
@@ -0,0 +1,10 @@
+require inc/xt_shared_env.inc
+
+EXTRA_OEMAKE += "PVRSRV_VZ_NUM_OSID=${XT_PVR_NUM_OSID}"
+DEPENDS += " gles-module-egl-headers wayland-native"
+RDEPENDS_${PN} += "python"
+
+do_install_append() {
```

Comment:
It seems you missed "sed -i 's/soc/passthrough/g' ${D}/etc/udev/rules.d/72-pvr-seat.rules" here
Could you re-check there else this one should be added

Reply:
Done